CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+4.50.0
```

Comment:
- do we need to add better exception handling in this scenarios? [#3883](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3883) and [#3614](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3614)
- [Bug] Logs are created even if logging is disabled [#3901](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3901) - did not see info on this
- [Bug] Managed Identity - Update logs or docs to clarify MI correlation ID is different than MSALs [#3908](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3908)
- [Request] WAM support for no scopes [#3675](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3675)